docs/examples/cookbook/drag-and-drop.md

```diff
@@ -87,7 +87,7 @@ local function Draggable(
 
 		BackgroundTransparency = 1,
 
-		Position = scope:Computed(function(use)
+		Position = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		Position = scope:Computed(function(use)
```

Reply:
reverted